lib/salesforceiq.js

```diff
@@ -447,6 +448,196 @@ var SalesforceIQ = (function() {
     makeRequest('lists/' + listId, req, cb);
   };
 
+  /**
+   * Format fields for list item to match SalesforceIQ API specs,
+   * using ids pulled from the associated list
+   * (c.f. https://api.salesforceiq.com/#/curl#documentation_lists_list-items_list-item-object)
+   *
+   * @param {string}                  listId  Reference id for the list containing these fields
+   * @param {Object.<string, string>} data    Simple key/value store of field data
```

Comment:
I'm not familiar with documenting param types like this. Are you trying to do something more like this?
http://usejsdoc.org/tags-param.html#parameters-with-properties